cea/optimization/slave/least_cost.py

```diff
@@ -1244,31 +1253,52 @@ def import_CentralizedPlant_data(fName, DAYS_IN_YEAR, HOURS_IN_DAY):
     """ import dataframes """
     # mass flows
 
-    # print fName
-    Q_DH_networkload = np.array(extract_csv(fName, "Q_DH_networkload", DAYS_IN_YEAR))
-    E_aux_ch = np.array(extract_csv(fName, "E_aux_ch", DAYS_IN_YEAR))
-
-    E_aux_dech = np.array(extract_csv(fName, "E_aux_dech", DAYS_IN_YEAR))
-    Q_missing = np.array(extract_csv(fName, "Q_missing", DAYS_IN_YEAR))
-    Q_storage_content_Wh = np.array(extract_csv(fName, "Q_storage_content_Wh", DAYS_IN_YEAR))
-    Q_to_storage = np.array(extract_csv(fName, "Q_to_storage", DAYS_IN_YEAR))
-    Q_from_storage = np.array(extract_csv(fName, "Q_from_storage_used", DAYS_IN_YEAR))
-    Q_uncontrollable = np.array(extract_csv(fName, "Q_uncontrollable_hot", DAYS_IN_YEAR))
-    E_PV_Wh = np.array(extract_csv(fName, "E_PV_Wh", DAYS_IN_YEAR))
-    E_PVT_Wh = np.array(extract_csv(fName, "E_PVT_Wh", DAYS_IN_YEAR))
+    # Extract data from all columns of the csv file to a pandas.Dataframe
+    result = pd.read_csv(fName, nrows=24 * DAYS_IN_YEAR)
+    Q_DH_networkload = (np.array(result['Q_DH_networkload'])[np.newaxis]).T
```

Comment:
Q_DH_networkload required, as this is the structure which was being passed previously and achieved using the transposing
[[ 237526.936812]
 [ 240580.214282]
 [ 244086.752048]
 ..., 
 [ 550347.814448]
 [ 325224.982603]
 [ 331011.737481]]

Value which we get just by using np.array(result['Q_DH_networkload'])
[ 237526.936812  240580.214282  244086.752048 ...,  550347.814448
  325224.982603  331011.737481]